lib/protocol/alertAccept.js

```diff
@@ -26,7 +26,7 @@ let alertAccept = function () {
         method: 'POST'
     }
 
-    return this.requestHandler.create(requestOptions)
+    return this.requestHandler.create(requestOptions, {dummy: 'dummy'})
```

Comment:
First of all why can't we just pass in an empty object (`{}`). Also as I said before, instead of changing each file, let's handle the payload one time in the requestHandler create function